tei_adapter/tei_id_extractor/docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
     ports:
       - "4566:4566"
   mysql:
-    image: "public.ecr.aws/docker/library/mysql:5.6"
+    image: "public.ecr.aws/docker/library/mysql:5.7"
```

Comment:
I bumped the version to 5.7 here because that's the version we're using in production, so it's better to run it locally too.